### PR TITLE
Update ascii art

### PR DESCRIPTION
**Pull request by osamuaoki**
I have created 2 candidate for the higher resolution opening logo.
Use of Braille fonts should be OK since AstroNvim already require NerdFont.

Please check both logos.  Please pick one and remove others.

Adding removed logo in user_examples may be good idea.

### CODE REVIEW

File: lua/configs/alpha.lua [hunk 1]
```diff
@@ -7,17 +7,33 @@ alpha.setup(astronvim.user_plugin_opts("plugins.alpha", {
...
+        "⠀⠀⠀⠀⠀⠀⣠⣾⣿⠟⠁⣼⣿⣿⣿⣿⣟⣉⠉⠛⣿⣿⡿⠛⢻⣿⣿⡿⠿⣿⣿⣿⠟⠛⣿⣿⣿⣿⣿⠈⢿⣿⣷⣸⣿⣿⣿⣷⡀⢿⣿⡿⣿⣿⡌⢿⣿⣿⠛⢿⣿⣿⡟⠻⣿⣿⣦⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⢠⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣰⣿⣿⠃⠀⣾⣿⡿⠀⢀⣿⣿⣿⠀⠀⣿⣿⣿⣿⣿⡄⠈⢻⣿⣿⣿⣿⣿⣿⣷⣸⣿⣧⢻⣿⣿⡌⣿⣿⣧⠀⠹⣿⣿⣆⠈⢻⣿⣷⡄⠀⠀⠀",
+        "⠀⠀⢀⣴⣿⡿⠿⠿⠿⢿⣿⣿⢿⣃⣈⣩⣿⣿⣿⣿⣿⣏⣀⣸⣿⣿⡇⠀⠸⣿⣿⣿⣄⣤⣿⣿⣿⣿⣿⡇⠀⠀⠻⣿⣿⣿⣷⠹⣿⣿⣿⣿⠀⢿⣿⣿⡘⣿⣿⣧⡀⠙⣿⣿⣦⠀⠙⣿⣿⣆⠀⠀",
+        "⢀⣴⣿⣿⠏⠀⠀⠀⠀⣾⣿⠁⣸⣿⣿⣿⡿⠟⠸⣿⣿⣿⣿⣿⣿⣿⠀⠀⠀⠻⣿⣿⣿⣿⣿⠟⢹⣿⣿⡇⠀⠀⠀⠹⣿⣿⣿⡆⠹⣿⣿⣿⠀⠘⣿⣿⣷⡘⣿⣿⣷⡀⠘⢿⣿⣷⡀⠈⢿⣿⣷⡄",
+        --
+        -- "⠀⠀⠀⠀⠀⣰⣿⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣤⣤⡄⠀⠀⢠⣤⡤⠀⠀⠀⠀⠀⠀⠀⠀⣰⣾⡷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀",
+        -- "⠀⠀⠀⢀⣼⡿⣿⣿⡆⠀⠀⠀⢀⣀⣀⡀⠀⣀⣿⣿⣁⡀⢀⣀⡀⢀⣀⠀⠀⠀⣀⣀⡀⠀⠀⠀⣼⣿⣿⣷⠀⠀⣸⣿⡇⣀⣀⡀⠀⠀⣀⣀⠀⣙⣋⠁⢀⣀⠀⢀⣀⡀⠀⢀⣀⡀",
+        -- "⠀⠀⢠⣾⡿⠁⣿⣿⡇⠀⢠⣾⡿⠿⠿⠏⠸⣿⣿⡿⠿⠀⣼⣿⣷⣿⡿⢀⣴⣿⠿⢿⣿⣷⠀⢰⣿⡟⢻⣿⣇⢠⣿⣿⠀⣿⣿⡇⢀⣼⣿⠋⢸⣿⡏⠀⣼⣿⡿⢿⣿⣿⡾⢿⣿⣿",
+        -- "⠀⢠⣿⣿⣷⣶⣿⣿⡇⠀⠸⢿⣿⣷⣶⡄⢀⣿⣿⠁⠀⢠⣿⣿⠁⠀⠀⣾⣿⡇⠀⢸⣿⡟⠀⣾⣿⠃⠈⣿⣿⣾⣿⠇⠀⢸⣿⣇⣾⡿⠁⠀⣿⣿⠃⢠⣿⣿⠀⢠⣿⣿⠀⢠⣿⣿",
+        -- "⣰⣿⡿⠉⠉⠉⢹⣿⣿⢠⣶⣶⣶⣿⠟⠀⠸⣿⣿⣶⠀⣼⣿⡇⠀⠀⠀⢿⣿⣷⣶⣿⠟⠁⢰⣿⡟⠀⠀⠸⣿⣿⡿⠀⠀⠸⣿⣿⡟⠀⠀⣸⣿⡟⠀⣾⣿⡇⠀⣾⣿⡇⠀⣼⣿⡇",
```
Comment (mehalter) [inline lua/configs/alpha.lua:36]: I think I like this one the best, could you update the commit to use this one and remove the last one. @kabinspace what do you think of this updated ASCII art?

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks so much for this contribution @osamuaoki ! These look great! I left a comment on the one I prefer and pinged @kabinspace to comment as well.

File: lua/configs/alpha.lua (hunk 1 above)
Comment (mehalter) [reply lua/configs/alpha.lua:36]: I chatted with @kabinspace and we like how bright and contrasted the current look is. It's a bit lighter which makes it harder to read. Could you try making the font a bit more bold? If it's not as contrasted we might stick with the current Ascii art and leave it a bit simpler. Thanks for investigating this though!

**Comment by kabinspace** [comment]
@osamuaoki I would like to stick with the old logo, for now, the old one seems to be more clean and bold, Thank you for your ideas nonetheless.

**Pull request closed without merging**